cli/src/cli.ts

```diff
@@ -302,6 +302,9 @@ async function parseCreateZTDFOptions(argv: Partial<mainArgs>): Promise<CreateZT
       throw new CLIError('CRITICAL', 'Invalid mimeType format');
     }
   }
+  if (argv.tdfSpecVersion) {
+    c.tdfSpecVersion = argv.tdfSpecVersion as never;
```

Comment:
I think I used a type guard in my branch? You should probably do that to give reasonable error messages here, instead of blind casting

Reply:
This is a great catch. I will update